client/modules/core/subscriptions.js

```diff
@@ -75,22 +76,35 @@ Tracker.autorun(() => {
   Subscriptions.Sessions = Meteor.subscribe("Sessions", Session.get("sessionId"));
 });
 
-// @see http://guide.meteor.com/data-loading.html#changing-arguments
 Tracker.autorun(() => {
-  let sessionId;
-  // we really don't need to track the sessionId here
-  Tracker.nonreactive(() => {
-    sessionId = Session.get("sessionId");
-  });
-  Subscriptions.Cart = Subscriptions.Manager.subscribe("Cart", sessionId, Meteor.userId());
+  Subscriptions.Account = Subscriptions.Manager.subscribe("Accounts", Meteor.userId());
```

Comment:
Well, I moved it because it is passing Meteor.userId(), a variable that reactively changes when logging in/out, but yet was not in an autorun. That means it wouldn't properly send an updated userId on logout/change. However...

Since you mentioned it I looked more closely at "Accounts" publication and it doesn't even use this value anywhere. So I pushed a commit moving this back and not passing anything.